app/policies/booking_policy.rb

```diff
@@ -0,0 +1,22 @@
+class BookingPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.all # index of bookings
```

Comment:
We should limit this to only renters.

Reply:
Many thanks for testing this thoroughly @carlnoval! I edited the index in booking_controller so that an alert appears and the user is redirected if he's a provider (And for users that are not signed-in, devise request login)